### PR TITLE
Riza messing up with HOOQ Hello CSV

### PR DESCRIPTION
**Pull request by rizafahmi**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict';
```
Comment (diorahman) [inline index.js]: This is interesting, please finish this one if you have time.

**Comment by diorahman** [comment]
Hi @rizafahmi thank you for submitting this PR! I believe it is easier for us to have a consolidated single commit with long comments attached to it. Please help us by doing so.

We also need to have your comments on the naive function. Thank you!

File: .jscsrc [hunk 2]
```diff
@@ -1,4 +1,3 @@
 {
-    "preset": "airbnb",
-    "validateIndentation": 4
-}
+	"preset": "airbnb"
```
Comment (diorahman) [inline .jscsrc]: Well we have chosen this pref. Please don't change it.

Comment (rizafahmi) [inline .jscsrc]: Sorry. I did run jscs setup. That's why it's changed

**Comment by rizafahmi** [comment]
@diorahman naive function is, well, naive :) and optimistic. There is no error exception. For example if the csv file doesn't exist, it will keep executing next code without helpful error message whatsoever.

Moreover, the for loop inside naive function is blocking. If the data(csv) get bigger, it'll takes more and more time to parse the file.

File: index.js (hunk 1 above)
Comment (rizafahmi) [inline index.js]: @diorahman ok I will. Need more time because testing in NodeJS is new experience for me. I usually do TDD in Python or Elixir :)

Comment (diorahman) [inline index.js]: Great. Thank you!

**Comment by rizafahmi** [comment]
@diorahman here you go the unit test: https://github.com/rizafahmi/hello-csv/blob/devel/test.js

Enjoy! ^_^

**Comment by diorahman** [comment]
Hi @rizafahmi thank you for your time taking this test and also having the follow up discussion! We decided to have internal discussion to decide. We will get back to you as soon as possible!

**Pull request closed without merging**